src/js/models/v2/aboutTheData/BaseReportingPeriodRow.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * BaseReportingPeriodRow.js
+ * Created by Lizzie Salita 12/8/20
+ */
+
+import { formatMoneyWithPrecision, formatNumber, calculatePercentage } from 'helpers/moneyFormatter';
+import { dateFormattedMonthDayYear, showQuarterText } from 'helpers/aboutTheDataHelper';
+
+const BaseReportingPeriodRow = {
+    populate(data, federalBudget) {
+        this._fiscalYear = data.fiscal_year || 0;
+        this._fiscalPeriod = parseInt(data.fiscal_period, 10) || 0;
+        this._budgetAuthority = data.current_total_budget_authority_amount || 0;
+        this._mostRecentUpdate = data.recent_publication_date || null;
+        // eslint-disable-next-line camelcase
+        this._missingTAS = data.tas_account_discrepancies_totals?.missing_tas_accounts_count || 0;
```

Comment:
Should we be more descriptive here? `missingTAS` leads me to question is this the count or total? 

Maybe `missingTASCount`?